Extended to include subscriptions to device events

danward79 opened the pull request:
Added module subscribe, tweaked readme and added some references

danward79 committed:
Early Subscribe Commit

Started to work on a subscription service. Also includes doc used for
reference, from UPnP Forum

danward79 committed:
More progress with listener

Added listener and XML is now unmarshal’d

danward79 committed:
Few comments and tweaks

danward79 committed:
Early bug

If the response body is not closed when a subscription is acknowledged
the following subscription fails.

danward79 committed:
Typo

danward79 committed:
Added Struct to use in storing subscriptions

danward79 committed:
Added Subscription Example

Work in progress. Likely to change

danward79 committed:
ManageSubscription added

Added manage subscription, to hand both subscription and
re-subscriptions after timeout

danward79 committed:
Event channel added for subscription notification

danward79 committed:
Tidy up

Still to sort manage subscriptions

danward79 committed:
Subscription Management Complete

Refactored (re)subscription management and tidied code up.

danward79 committed:
Typo!

danward79 committed:
Create .travis.yml

danward79 commented:
Hi@savaki, will you accept pull request?

savaki commented:
Just seeing this.  Gimme a day or two to check out your pull request.

M

On Mon, Aug 4, 2014 at 3:10 PM, Dan notifications@github.com wrote:

> Hi@savaki, will you accept pull request?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/savaki/go.wemo/pull/1#issuecomment-51125076.

danward79 committed:
Fixed a couple of bits and tidied up

danward79 committed:
Delete .travis.yml

danward79 committed:
Catch failure when network is down

danward79 committed:
reduced logging

danward79 committed:
Update README.md

lindsaymarkward committed:
added urn for netcam so it finds NetCam camera

Commit:
Merge pull request #1 from lindsaymarkward/master

Update library and add support for finding NetCam camera

jonseymour committed:
Update context package referenence.

jonseymour committed:
Merge branch 'fix-net-context-reference'

jonseymour committed:
Add support for insight device type.

jonseymour committed:
Allow calls to services otherthan basic event.

jonseymour committed:
Add support for extracting the power reading from insight devices.

Signed-off-by: Jon Seymour <jon@wildducktheories.com>

danward79 committed:
ready for merge

danward79 committed:
Merge branch 'catchup'

danward79 committed:
Catchup & Clean up

Merged latest from another fork. There are a lot of golint warnings so
have been clearing those.

danward79 committed:
More golint warning fixes

danward79 committed:
Extract UDN, to support bridge devices

danward79 committed:
Bridge device discovery

Added support for finding bridge devices and fetching bridge device
info. Updated CLI util

danward79 committed:
tidy up

danward79 committed:
Bulb control added

Added control of demo end device bulbs (grouped also) and did a little
cleaning up.

Updated cli util also.

danward79 committed:
Updated to get bulb status

Updated to get bulb status and clear up a pesky golint shadow error

danward79 committed:
Tidy up

danward79 committed:
Tidy up

danward79 committed:
Clean up after linter

danward79 committed:
Further clean ups

danward79 committed:
Added link subscription

danward79 committed:
Additional tests and travis integration

cml

readme link

readme

test lint errors

danward79 committed:
adding tests

danward79 committed:
cml

danward79 committed:
readme link

danward79 committed:
readme

danward79 committed:
test lint errors

danward79 committed:
Merge branch 'master' of https://github.com/danward79/go.wemo

danward79 committed:
Addition of basic URN devices

danward79 committed:
Merge branch 'Find-Fix-for-Poor-Discovery-Performance'

danward79 committed:
Ignore binary

danward79 committed:
Readme update

danward79 committed:
Added wrapper SetState

danward79 committed:
Correction to request close
